src/io/vtk.rs

```diff
@@ -335,7 +335,11 @@ where
     /// # Panics
     /// Panics if the number of attribute entries is not divisible by the number of points,
     /// if there are 0 components per vector or if there are more than 3 components per vector.
-    pub fn with_point_vector_attributes(self, name: impl Into<String>, attributes: &[T]) -> Self {
+    pub fn with_point_vector_attributes<S: Scalar + Zero + ToPrimitive>(
+        self,
+        name: impl Into<String>,
+        attributes: &[S],
+    ) -> Self {
```

Comment:
Just to clarify: the case here is to be able to for example attach an integer attribute to a mesh with `T = f64`? This makes a lot of sense of course!

Reply:
Exactly.